tests/integration/relations/test_database.py

```diff
@@ -239,6 +260,9 @@ async def test_relation_creation(ops_test: OpsTest):
 
         await ops_test.model.wait_for_idle(apps=APPS, status="active")
 
+    relation_data = await get_relation_data(ops_test, APPLICATION_APP_NAME, "database")
+    assert not set(["password", "username"]) <= set(relation_data.keys[0])
```

Comment:
NOTE: This test will FAIL unitl https://github.com/canonical/mysql-test-app/pull/14 is merged and populated to charmhub.